jobs/tasks.py

```diff
@@ -28,14 +31,15 @@ def prediction(job,model):
         raise e
     finally:
         job.save()
-
+        
 def calculate(job,model):
     """ Main entry method for calculations"""
     #print("Start job {} with {}".format(job['type'], get_run(job)))
```

Comment:
Did you comment out these print loggers? I'd rather like them to be working, same in `training`.